tests/test_content.py

```diff
@@ -421,6 +395,77 @@ def test_datetime():
     assert new_result == '2001-02-03 04:05'  # output
 
 
+def test_param_align():
+    """
+    # Parametrized Formats
```

Comment:
Please write "Parametrized formats" instead of "Parametrized Formats" here to be more consistent with other headings 😃